src/test/java/com/snowflake/kafka/connector/internal/SinkServiceIT.java

```diff
@@ -763,14 +763,14 @@ public void testCleanerRecover() throws Exception
 
     System.out.println("break connection");
     doThrow(SnowflakeErrors.ERROR_2001.getException()).when(spyConn).purgeStage(anyString(), anyList());
-    // Sleep two minutes so that cleaner encounters two exceptions.
-    Thread.sleep(SnowflakeSinkServiceV1.CLEAN_TIME * 2);
+    // Sleep 6 minutes so that cleaner encounters 6 exceptions. Just to make sure cleaner restart is triggered
+    Thread.sleep(6 * 60 * 1000);
 
     System.out.println("recover connection");
     doCallRealMethod().when(spyConn).purgeStage(anyString(), anyList());
 
-    // read ingestHistory
-    Thread.sleep(480000);
+    // Total sleep time is 10 minutes to test read ingestHistory
+    Thread.sleep(4 * 60 * 1000);
```

Comment:
Ia thi sten minutes or four minutes

Reply:
Here we first sleep 6 minutes with purge() mocked to throw error, then we recover purge() and wait for 4 minutes. So the total sleep time is 10 minutes.